src/crop.sol

```diff
@@ -159,7 +164,9 @@ contract CropJoin {
             vat.slip(ilk, msg.sender, -int256(wad));
 
             total = sub(total, wad);
-            stake[msg.sender] = sub(stake[msg.sender], wad);
+            uint256 _stake = stake[msg.sender];
```

Comment:
Can you clarify why we would need it in `exit`? Shouldn't be enough with the overwriting already done in `harvest`?

Reply:
Previously we were requiring the user have the necessary stake. This will remove the stake only if it is available. Once we break this stake = gems invariant though harvest needs the additional check to ensure the gems are there.